\chapter*{\fontsize{36}{50}\selectfont Part 4\, \newline {\fontsize{18}{32}\selectfont Reflection and discussion}}
\chaptermark{Part 4}
\label{chap:part4}


{ \Large \leftwatermark{
		\put(-67,-66.5){ 1 }
		\put(-67,-91.5){ 2 }
		\put(-67,-116.5){ 3 }
		\put(-67,-141.5){ 4 }
		\put(-67,-166.5){ 5 }
		\put(-67,-191.5){ 6 }
		\put(-67,-216.5){ 7 }
		\put(-67,-241.5){ 8 }
		\put(-76.5,-275){\includegraphics[scale=0.8]{img/thumbindex.eps}} \put(-67,-266.5){ {\color{white} 9 }}
		\put(-67,-291.5){ 10 }
		\put(-67,-316.5){ 11 }
		
	} \rightwatermark{
		\put(350.5,-66.5){ 1 }
		\put(350.5,-91.5){ 2 }
		\put(350.5,-116.5){ 3 }
		\put(350.5,-141.5){ 4 }
		\put(350.5,-166.5){ 5 }
		\put(350.5,-191.5){ 6 }
		\put(350.5,-216.5){ 7 }
		\put(350.5,-241.5){ 8 }
		\put(346.5,-275){\includegraphics[scale=0.8]{img/thumbindex.eps}} \put(350.5,-266.5){ {\color{white} 9 }}
		\put(350.5,-291.5){ 10 }
		\put(350.5,-316.5){ 11 }
}}


\newpage

\noindent The introduction of next-generation sequencing (NGS) techniques has revolutionized the field of genomics. 
It enabled the “1000 dollar genome” \cite{Rincon_2014} and the analysis of whole panels of genes associated with a specific phenotype. 
It has also enabled large-scale bulk and single-cell RNA analyses \cite{van_der_Wijst_2018}, as well as epigenetics analysis \cite{Jiang_2018,Le_Dily_2017,Andrews_2018}. 
In this thesis we have shown how NGS techniques can be applied in different types of DNA analysis, focusing on detection of germline variants and somatic chromosomal translocations and non-invasive prenatal testing (NIPT). 
In addition to creating optimized laboratory protocols for NGS sample preparation, we have created a number of new algorithms to extract biologically relevant information from the data produced. 
These allow us to look through the noise created during the laboratory process – such as batch-effects, PCR-efficiency, capturing efficiency and effects of combining primers in a multiplex – and the biological noise present in the sample itself, such as the presence of non-aberrant cells in detection of somatic chromosomal translocations or the presence of a high percentage of maternal cell-free DNA compared to cell-free fetal DNA in the mother’s blood. 

The growing list of NGS applications, and their use in diagnostics and research, have shown that NGS can already compete with or improve on conventional techniques for genetic analysis, most notably Sanger sequencing. 
In the years to come, sample preparation methods, sequencing strategies and analysis algorithms will develop further, and this will create opportunities to fill in the current gaps in NGS that lead to conventional techniques still being the preferred approach for some questions, such as structural variant calling and variant detection in extended regions that appear more than once in the genome (although those techniques each have their own gaps). 

However, the methodology of NGS techniques, as well as the social effects of its comprehensive results, requires more discussion. 
I will therefore use a scientific philosophical perspective framed by the three questions posed by Immanuel Kant in his \textsl{Kritik der reinen Vernunft} published in 1781/1787: “what can I know?”, “what should I do?” and “what may I hope?” \cite{Kant_1781a}[p. 728]. 
My application of these questions is of a more profane nature than their original intent. 
The first question I will discuss, in chapter 9, is \textsl{‘What can I know?’}. 
I will use this question to reflect on the nature of the data analyzed. 
The discussion in this chapter will remain on an abstract level, and the practical solutions or methods to address the issues broached here will be discussed later. 
In chapter 10 I will explore answers to Kant’s second question \textsl{‘What should I do?’} to discuss ethical issues of genetic analysis, in particular the analyses introduced in this thesis. 
Finally, in chapter 11 I will use the question \textsl{‘What may I hope?’} to discuss how to fill in remaining gaps regarding variant detection using existing techniques, deliberate upon future perspectives and look forward towards yet another next generation of sequencing techniques.
